### PR TITLE
add enigma972 user bundle

### PR DESCRIPTION
**Pull request by enigma972**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

File: enigma972/user-bundle/0.3/config/packages/user.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+user:
```
Comment (Nyholm) [inline enigma972/user-bundle/0.3/config/packages/user.yaml:1]: Lets change the name of this file to `enigma972_user.yaml`

File: enigma972/user-bundle/0.3/config/routes/user.yaml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+user:
```
Comment (Nyholm) [inline enigma972/user-bundle/0.3/config/routes/user.yaml:1]: Same with this file

**Comment by Nyholm** [comment]
It seams like Travis cannot install the bundle. Can you make sure at least one Travis is green?

File: enigma972/user-bundle/0.3/config/packages/user.yaml (hunk 1 above)
Comment (enigma972) [reply enigma972/user-bundle/0.3/config/packages/user.yaml]: It's ok.

**Comment by enigma972** [comment]
> It seams like Travis cannot install the bundle. Can you make sure at least one Travis is green?

Now yes, a travis is green with Symfony ^4.4

**Comment by Nyholm** [comment]
@symfony-flex-server review please

**Review by Nyholm — APPROVED** [review]
Thank you

**Pull request merged**